platforms/chip/README.md

```diff
@@ -11,6 +11,9 @@ For documentation about the C.H.I.P. platform click [here](http://docs.getchip.c
 go get -d -u gobot.io/x/gobot/... && go install gobot.io/x/gobot/platforms/chip
 ```
 
+Note that PWM might not be available in your kernel and you might need to load the right overlays
+to expose PWM on the PWM0 pin.
+
```

Comment:
This is where instructions on how to install the dtc overlay on CHIP using Gort should go :)